src/safeds/data/tabular/containers/_table.py

```diff
@@ -867,9 +867,9 @@ def rename_column(self, old_name: str, new_name: str) -> Table:
         new_df.columns = self._schema.column_names
         return Table._from_pandas_dataframe(new_df.rename(columns={old_name: new_name}))
 
-    def replace_column(self, old_column_name: str, new_column: Column) -> Table:
+    def replace_column(self, old_column_name: str, new_columns: list[Column]) -> Table:
```

Comment:
I know this what not specified in the original issue, but I think it would make sense here to accept a variable number of args for the new columns (with the * operator), as this will provide further comfort to the user, who then is able to use the function in several ways, without putting one ore more single columns in a list, e.g.: replace_column(old, new_column) / replace_column(old, new_column1, new_column2, new_column3) / replace_column(old, *new_columns)

Reply:
added the options to give a Column or a Table as parameter. I think the option with *new_columns would make it more confusing because the function call could get unnecessarily big.